File: 프로그래머스/Lv.0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        double discountPrice = price;
        if(price >= 500000){
            discountPrice = price * 0.8;
        }else if(price >= 300000){
            discountPrice = price * 0.9;
        }else if(price >= 100000){
            discountPrice = price * 0.95;
        }
        return (int)discountPrice;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.js
function solution(price) {
    let discount = 0;
    if(price >= 500000){
        discount = price * 0.8;
    }else if(price >= 300000){
        discount = price * 0.9;
    }else if(price >= 100000){
        discount = price * 0.95;
    }else{
        discount = price;
    }
 
    return Math.floor(discount);
}
